### ISSUE TITLE
adding wiskis query support 

### ISSUE THREAD

**Issue by petersilva** [MetPX/sarracenia#758]
This is a copy of an internal to Government of Canada request... there is more detail there... Briefly:

request from client:

> We would like your assistance in acquiring data from the Ontario Power Energy. They changed their protocol of transfer to the new WISKI system. 
> We would like to do tests with a sample of data before to switch to the new system.

This is a reference to: https://www.kisters.net/wiski/

and there are some python bindings here: https://pypi.org/project/kiwis-pie/

and we also have a VB sample from OPG (omitted because it mixes credentials and code.)

A substantial complication is that one uses a username to obtain a token, and then do the queries using the tokens for authentication.

@Rehouma63 provided an initial proof of concept standalone python script that obtained a sample time series.  I have
re-worked it to fit into sarracenia for eventual deployment.

**Comment by petersilva** [comment]
@Rehouma63 finished his term, so I picked up where he had gotten to and brought it a little further.
Here is a sample file being produced:  

fractal% more /tmp/wiski/ts_23755_2023-09-19h01:19:44.629537+00:00_2023-09-20h01:19:44.629537+00:00.csv
Timestamp,Value
2023-09-19 00:00:00-05:00,390.07
2023-09-19 00:01:00-05:00,390.08
2023-09-19 00:02:00-05:00,390.1
2023-09-19 00:03:00-05:00,390.1
2023-09-19 00:04:00-05:00,390.11
...
(time series covers 24 hours and has a single value in it: Air Temperature)

things that puzzle me:

* do people want more than one value/station? which ones?
* the data seems to only come up for one station, no matter what I do.
* The token code works, but I had it only be invoked when we encountered a 403 (permission denied) and we never do, and it works anyways... so the token auth stuff is pointless?
